test/test_nestedtensor.py

```diff
@@ -3210,7 +3210,7 @@ def _validate_nt(self, nt, tensor_list, device, dtype, requires_grad):
         self.assertTrue(isinstance(nt.shape[1], torch.SymInt))
         self.assertEqual(nt.shape[2:], first_t.shape[1:])
 
-    @torch._dynamo.config.patch(suppress_errors=True)
+    @skipIfTorchDynamo("Dynamo type of torch.SymInt returns int")
```

Comment:
What exactly does this error look like?

Reply:
We had discussed this at https://github.com/pytorch/pytorch/pull/113009#discussion_r1384010262